README.md

```diff
@@ -90,7 +90,7 @@ For more information, check out the documentation and tutorials at [zuko.readthe
 | `UNAF`  | 2019 | [Unconstrained Monotonic Neural Networks](https://arxiv.org/abs/1908.05164) |
 | `CNF`   | 2018 | [Neural Ordinary Differential Equations](https://arxiv.org/abs/1806.07366) |
 | `GF`    | 2020 | [Gaussianization Flows](https://arxiv.org/abs/2003.01941) |
-| `BPF`   | 2020 | [Bernstein-Polynomial Normalizing Flows](https://arxiv.org/abs/2204.13939) |
+| `BPF`   | 2020 | [Bernstein-Polynomial Normalizing Flows](https://arxiv.org/abs/2004.00464) |
```

Comment:
Should we cite to Sick et al. (2020) in the transformation and flow as well? I cited Arpogaus et al. (2022) as it was the first to use the term "Berstein-Polynomial Normalizing Flows".

Reply:
We should cite Sick et al. (2020) here as well since this was the first time Bernstein Polynomials have been used with NN to control a 1D flow.

Reply:
Yes, that's why I changed it to Sick at al. here, but kept both references in the doc string.